src/main/resources/application.properties

```diff
@@ -34,6 +28,7 @@ lms.user.update=private/user/v1/update
 lms.user.read=private/user/v1/read/userIdValue
 lms.user.migrate=private/user/v1/migrate 
 lms.assign.role=private/user/v1/assign/role
+lms.system.settings.path=v1/system/settings/get/wfProfileServiceConfig
```

Comment:
change the variable name -- to mention that this is wfProfileService config path